manual.typ

```diff
@@ -441,6 +440,25 @@ for-each-anchor("demo", (name) => {
 })
 ```
 
+== Layers
+
+You can use layers to draw elements below or on top of other elements by using layers
+with a z-index. When rendering, all draw commands are sorted by their z-index (0 being
+the default).
```

Comment:
Can we stick with the `layers` name? `z-index` could get confusing when we also allow a `z` component.

Reply:
I don't see these changes?

Reply:
Oops, I had changed the code comment but missed the docs :)